server/config/config.go

```diff
@@ -971,6 +973,7 @@ func (man Manager) addConfigs() {
 	man.addConfigDuration("redis.conn_wait_timeout", 0, "Redis maximum amount of time to wait for a connection if the maximum is reached (0 for no wait)")
 	man.addConfigDuration("redis.write_timeout", 10*time.Second, "Redis maximum amount of time to wait for a write (send) on a connection")
 	man.addConfigDuration("redis.read_timeout", 10*time.Second, "Redis maximum amount of time to wait for a read (receive) on a connection")
+	man.addConfigDuration("redis.live_query_cache_expiration", 1*time.Second, "Time to cache Redis live query keys in memory")
```

Comment:
Any reason this is configurable? Setting this to some high (untested) value could cause issues.

Reply:
good call, i removed the config